### PR TITLE
Add toggles to XP Orb hovered line items display

### LINKED ISSUE #9497: Add toggles to XP Orb hovered line items display
Additional context

on the xp orb we should be able to see what we want to see what I mean is right now whenever we move out mouse on top of the xp orb we see this which is located on the left side of the image underneath.
image
image

but I want to pick what I want to see out of all the options which are;

**name of skill, current XP, Kills Left, XP Left, XP per hour.**
we should be able to pick which ones we want to see out of the list above.
so, for example, you can make a plugin at the 'configuration' which is called 'XP Orb'
and in that XP Orb you can have it saying
do you want to see **name of skill,**
do you want to see **current XP,**
do you want to see **Kills Left,**
do you want to see **XP Left,**
do you want to see **XP per hour,**

or you don't have to have it saying 'do you want to see' you can just have the enable/disable plugin which is this
image

example of this 
name of skill image
current XP image
Kills Left image
XP Left image
XP per hour image

### PR DESCRIPTION
**Pull request by Darkkgreen**
Just added conditionals to control what is displayed inside the tooltip box.

Closes #9497

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 1]
```diff
@@ -44,11 +44,44 @@ default boolean enableTooltips()
...
+		description = "Shows XP Left inside the globe tooltip box",
+		position = 1
+	)
+	default boolean showXPLeft()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "showActionsLeft",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: Use one tab for indentation

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 2]
```diff
@@ -44,11 +44,44 @@ default boolean enableTooltips()
...
+			description = "Shows the number of actions left inside the globe tooltip box",
+			position = 2
+	)
+	default boolean showActionsLeft()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "showXPHourLeft",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: and here

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 3]
```diff
@@ -44,11 +44,44 @@ default boolean enableTooltips()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "showXPLeft",
+		name = "Show XP Left",
+		description = "Shows XP Left inside the globe tooltip box",
+		position = 1
+	)
+	default boolean showXPLeft()
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: generally we would name this like: `showXpLeft`

**Comment by Darkkgreen** [comment]
I'll fix it later tonight

**Comment by Darkkgreen** [comment]
Done

**Comment by Adam-** [comment]
Additionally, "Xp/hr left" isn't a thing and has no meaning

**Comment by Darkkgreen** [comment]
Oopsies

**Comment by xDemoN** [comment]
All of the third config item should be "showXpHour"

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java [hunk 4]
```diff
@@ -44,11 +44,44 @@ default boolean enableTooltips()
...
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "showXPHour",
+		name = "Show XP/hr",
+		description = "Shows XP per hour inside the globe tooltip box",
+		position = 3
+	)
+	default boolean showXPHourLeft()
```
Comment (xDemoN) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesConfig.java]: Missed this one

**Pull request merged**